### PR TITLE
manylinux2010

### PR DESCRIPTION
**Pull request by ijl**
This enables what was mostly there already, fixing #69. I built a manylinux2010 wheel locally and it installs fine with pip 19.0.

### CODE REVIEW

File: src/build_context.rs [hunk 1]
```diff
@@ -151,9 +151,21 @@ impl BuildContext {
...
+                        auditwheel_rs(&artifact, MANYLINUX1)
+                            .context("Failed to ensure manylinux compliance")?;
+                    }
+                    Manylinux::Manylinux2010 => {
+                        auditwheel_rs(&artifact, MANYLINUX2010)
+                            .context("Failed to ensure manylinux compliance")?;
+                    }
+                    _ => {}
+                }
+            }
```
Comment (konstin) [inline src/build_context.rs]: Please move the above into a method

File: src/build_context.rs [hunk 2]
```diff
@@ -204,9 +216,21 @@ impl BuildContext {
...
```
Comment (konstin) [inline src/build_context.rs]: This should also use the method

**Comment by konstin** [comment]
Thank you! Please add that method, otherwise the PR looks good.

**Comment by ijl** [comment]
Ok.

**Pull request merged**